main.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable linebreak-style */
+
+const list = document.getElementById('list');
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const buttonClass = document.querySelector('.buttonClass');
+const books = [];
+
+function saveBooks(newBooks) {
+  // books = newBooks;
+  localStorage.setItem('books', JSON.stringify(newBooks));
+}
+
+function addItem(title, author) {
+  const reloadBooks = JSON.parse(localStorage.getItem('books'));
```

Comment:
- [ ] Kindly change these functions to es6 arrow functions, as required under the [JavaScript best practices guide](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) to use es6 syntaxes.